src/Diagnostics.RuntimeHost/appsettings.json

```diff
@@ -104,6 +104,12 @@
     "Enabled": true,
     "MdmCertSubjectName": ""
   },
+  "ContainerAppsMdm": {
```

Comment:
Michi, does this work involve setting up a new cert in KV to talk to MDM Container Apps endpoint?

Reply:
Yes, it does. And I already finished installing the new cert to all the runtime host apps.